phalcon/Http/Request.zep

```diff
@@ -206,13 +205,13 @@ class Request implements RequestInterface, InjectionAwareInterface
 
         if fetch contentType, _SERVER["CONTENT_TYPE"] {
             return contentType;
-        } else {
-            /**
-             * @see https://bugs.php.net/bug.php?id=66606
-             */
-            if fetch contentType, _SERVER["HTTP_CONTENT_TYPE"] {
-                return contentType;
-            }
+        }
+
+        /**
+         * @see https://bugs.php.net/bug.php?id=66606
```

Comment:
Current bug was fixed in `5.5.8` which Phalcon 4 doesn't support...

Reply:
Thanks for the heads up. I've removed it in 7c81838